modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -0,0 +1,532 @@
+/* global __filename, RTCIceGatherer, RTCIceTransport */
+
+import { getLogger } from 'jitsi-meet-logger';
+import yaeti from 'yaeti';
+import { InvalidStateError } from './errors';
+
+const logger = getLogger(__filename);
+
+// Whether we want to mimic the new Promises based RTCPeerConnection or use the
+// old callbacks based one.
+const USE_PROMISES = false;
+
+/**
+ * RTCPeerConnection shim for ORTC based endpoints (such as Edge).
+ *
+ * The interface is based on the W3C specification of 2015, which matches
+ * the implementation of Chrome nowadays:
+ *
+ *   https://www.w3.org/TR/2015/WD-webrtc-20150210/
+ */
+export default class ortcRTCPeerConnection extends yaeti.EventTarget {
+    /**
+     */
+    constructor(pcConfig) {
+        super();
+
+        logger.debug('constructor() pcConfig:', pcConfig);
+
+        // Closed flag.
```

Comment:
Really? I strongly prefer having them shorted "logically"... Is that a style strong requirement?